input/fsh/SD_Patient.fsh

```diff
@@ -6,3 +6,16 @@ Description: "A patient who has been diagnosed with or is receiving medical trea
 * ^extension[FMM].valueInteger = 5
 * deceased[x] MS
 * . MS
+
+Profile: HistoryOfMalignantNeoplasm
+Parent: USCoreConditionProblemsHealthConcernsProfile
+Id: mcode-history-of-malignant-neoplasm
+Title: "History of Malignant Neoplasm"
+Description: "Records the existence of a past episode of cancer, for the purpose of long term management and tracking."
+* code from HistoryOfMalignantNeoplasmVS
+
+ValueSet: HistoryOfMalignantNeoplasmVS
+Id: mcode-istory-of-malignant-neoplasm-vs
```

Comment:
@markkramerus - is this a typo? It has `mcode-istory-of-malignant-neoplasm-vs`

Reply:
Fixed